jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -178,6 +179,9 @@ export default function EmailSection() {
           color='blue'
           type='frame'
           onClick={setNewUserEmail.bind(setNewUserEmail, email.newEmail)}
+          // quick simple subtle email validation to avoid complete accidents
+          // a toast showing any API error feedback would be nicer
+          isDisabled={!/[^@]+@[^@]+\.[^@]+/.test(email.newEmail)}
```

Comment:
Found this problem during testing, not super relevant but thought I'd slip it in while I'm here.